Sources/Update/PackagesDirectory.swift

```diff
@@ -0,0 +1,27 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright 2015 - 2016 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import struct PackageDescription.Version
+import class PackageDescription.Package
+import PackageType
+import Utility
+import Get
+
+struct PackagesDirectory {
```

Comment:
I find it confusing to have two `PackagesDirectory` types in the same project with very similar purposes, but not in the same module. Should this be merged with `Get.PackagesDirectory`?